designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview-field-recon.component.js

```diff
@@ -239,17 +239,12 @@ class PublicationPreviewFieldReconCtrl {
     }
 
     showAuthor(author) {
```

Comment:
We should consider creating a higher order class for common publication action components, or at least a utility component to handle these shared functions across components.